src/backend/executor/spi.c

```diff
@@ -2278,6 +2279,17 @@ _SPI_execute_plan(SPIPlanPtr plan, ParamListInfo paramLI,
 
 			dest = CreateDestReceiver(canSetTag ? DestSPI : DestNone);
 
+PG_TRY();
```

Comment:
Why did you keep the indentation when you added PG_TRY, but when you removed PG_TRY you shifted the indentation?

Reply:
I didn't change indentation in the place where I added PG_TRY to keep diff at minimum, that's the way they use in gp frequently. Also, I changed indentation in the place where I removed PG_TRY because it would look weird if I'd keep it as it was